wgsl/wgsl.recursive.bs.include

```diff
@@ -0,0 +1,693 @@
+<div class='syntax' noexport='true'>
+  <dfn for='right_syntax'>access_mode</dfn>:
+
+ | `'read'`
+
+ | `'read_write'`
+
+ | `'write'`
+</div>
+
+<div class='syntax' noexport='true'>
+  <dfn for='right_syntax'>additive_operator</dfn>:
+
+ | [=syntax/minus=]
+
+ | [=syntax/plus=]
+</div>
+
+<div class='syntax' noexport='true'>
+  <dfn for='right_syntax'>address_space</dfn>:
```

Comment:
What does `right_syntax` mean?

Reply:
It's a tag, other than "syntax",  for differentiating kinds of definitions.

e.g. "expression" has a main body definition, a definition "for" syntax (the main grammar), and "right syntax" which is for right-recursive grammar, or recursive descent.  I guess I should change it to recursive descent...

Reply:
<img width="259" alt="Screen Shot 2022-08-15 at 2 35 48 PM" src="https://user-images.githubusercontent.com/13054574/184695222-a19219d0-1d1a-4553-923d-407e212d9b4d.png">

Reply:
But, why is the definition for `right_syntax` "address_space" ?

Reply:
The "for" tag is "in what context is this definition being made".
So it's address_space "for" recursive descent grammar.

Reply:
This is what "address_space" looks like in the index:

<img width="230" alt="Screen Shot 2022-08-15 at 3 15 32 PM" src="https://user-images.githubusercontent.com/13054574/184701815-285f3c9a-8a40-4b99-9f4a-0d3c6fd9c8c1.png">

Reply:
I see, interesting.